### DISCUSSION TITLE
How to use different db connection for compiling and running code

### DISCUSSION THREAD

**Discussion by ccecvb** [vscode-abl/vscode-abl#138] [Q&A]
For compile performance reasons, we connect to read-only single user databases to compile.
`openedge-project.json` contains those `-RO` connections.

Can I specify an alternate db connection to be used when running code ?

I've been looking at the "procedures" in the config, but I don't think there is something there that can be used.

**Comment by gquerret** [comment]
There's no option to run code with a different set of DB (or set of options). As this is not very high in the list of priorities, I'd recommend using VSCode extensions for that, such as [Code Runner](https://marketplace.visualstudio.com/items?itemName=formulahendry.code-runner).
